src/categories/create.js

```diff
@@ -39,15 +40,18 @@ module.exports = function(Categories) {
 
 				var defaultPrivileges = ['find', 'read', 'topics:create', 'topics:reply'];
 
-				async.series([
```

Comment:
Yes, this ensures that the series is executed over the modified category data returned by the filter.